src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown

```diff
@@ -310,29 +369,35 @@ I would recommend you check out the official [Bluetooth debug page](https://site
 
 ## What's next
 
-As the Web Bluetooth API implementation is not complete yet, here's a sneak
-peek of what to expect in the coming months:
+Check the [browser and platform implementation
+status](https://github.com/WebBluetoothCG/web-bluetooth/blob/gh-pages/implementation-status.md)
+first to know which parts of the Web Bluetooth API are currently being implemented.
+
+As it is still not complete yet, here's a sneak peek of what to expect in the
```

Comment:
'As it is still not complete yet' > 'Though it's still incomplete'. Or something similar.